### PR TITLE
Create config store for static content

### PR DESCRIPTION
**Pull request by magmat88**

### CODE REVIEW

File: src/app/components/aboutPage/aboutPage.component.tsx [hunk 1]
```diff
@@ -1,248 +1,38 @@
...
-                            The space for living and working on space station is
-                            larger than a six-bedroom house
-                        </div>
-                    </a>
-                </li>
-            </ul>
-
-            <div className="about-page-item-title">
+            </section>
+            <List listItems={config.staticContent.aboutPage.facts} />
```
Comment (earendillo) [inline src/app/components/aboutPage/aboutPage.component.tsx:20]: List is a very generic name, does this component will be able to handle ANY list passed as listItems?

File: src/app/components/list/list.component.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+import { ListItem } from '../../modules/config/config.state';
+import { ListItemTile } from '../listItem/listItem.component';
+
+interface ListProps {
+    listItems: Array<ListItem>;
```
Comment (earendillo) [inline src/app/components/list/list.component.tsx:7]: List is too generic name, I'd suggest to use more specific name. E.g. staticContentItems if there would be a some base interface for staticContentItem which could be extended

File: src/app/modules/config/config.events.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+export const ConfigInitEvent = Symbol('Config Init');
+
+export type ConfigInitPayload = {
+    aboutPage: any;
```
Comment (earendillo) [inline src/app/modules/config/config.events.ts:4]: please create interface for aboutPage, this is quite complex object

File: src/app/modules/config/config.state.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
+import { CONFIG_STORE_KEY } from './config.module';
+
+export interface ListItem {
```
Comment (earendillo) [inline src/app/modules/config/config.state.ts:3]: ListItem is too generic

File: src/app/config/staticContent.ts [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+            },
+            {
+                icon: reports,
+                iconClassName: 'icon-medium',
+                alt: 'Graph on monitor icon',
+                iconSource:
+                    'https://pixabay.com/pl/vectors/p%c5%82aski-projekt-symbol-ikona-www-2126876/',
+                href: null,
+                content: 'Latest data from space stations and missions',
+            },
```
Comment (earendillo) [inline src/app/config/staticContent.ts:82]: it's good practice to do not leave a trailing comma in the end of the list. I think prettier could be configured to keep that rule.

**Review by earendillo — CHANGES_REQUESTED** [review]

File: src/app/components/list/list.component.tsx (hunk 2 above)
Comment (magmat88) [reply src/app/components/list/list.component.tsx:7]: changed

File: src/app/modules/config/config.state.ts (hunk 4 above)
Comment (magmat88) [reply src/app/modules/config/config.state.ts:3]: changed

File: src/app/config/staticContent.ts (hunk 5 above)
Comment (magmat88) [reply src/app/config/staticContent.ts:82]: done

File: src/app/components/aboutPage/aboutPage.component.tsx (hunk 1 above)
Comment (magmat88) [reply src/app/components/aboutPage/aboutPage.component.tsx:20]: done

File: src/app/modules/config/config.events.ts (hunk 3 above)
Comment (magmat88) [reply src/app/modules/config/config.events.ts]: created type AboutPage

**Review by earendillo — APPROVED** [review]

File: src/app/components/aboutPage/aboutPage.component.tsx [hunk 6]
```diff
@@ -1,248 +1,41 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+
```
Comment (earendillo) [inline src/app/components/aboutPage/aboutPage.component.tsx:2]: why empty line? remove it

File: src/app/components/aboutPage/aboutPage.component.tsx [hunk 7]
```diff
@@ -1,248 +1,41 @@
...
 import './aboutPage.component.scss';
-import blog from '../../images/blog.svg';
-import fact from '../../images/fact.svg';
-import heart from '../../images/heart.svg';
-import interesting from '../../images/interesting.svg';
-import news from '../../images/news.svg';
-import programmer from '../../images/programmer.svg';
-import quality from '../../images/quality.svg';
-import reports from '../../images/reports.svg';
+import { StaticContentList } from '../staticContentList/staticContentList.component';
```
Comment (earendillo) [inline src/app/components/aboutPage/aboutPage.component.tsx:6]: please keep order of imports consistent

**Review by earendillo — CHANGES_REQUESTED** [review]

File: src/app/components/aboutPage/aboutPage.component.tsx (hunk 6 above)
Comment (magmat88) [reply src/app/components/aboutPage/aboutPage.component.tsx]: removed

File: src/app/components/aboutPage/aboutPage.component.tsx (hunk 7 above)
Comment (magmat88) [reply src/app/components/aboutPage/aboutPage.component.tsx:3]: sorted imports

**Review by earendillo — APPROVED** [review]

**Pull request merged**